docs/src/prerequisites.md

```diff
@@ -1,5 +1,10 @@
 ## Pre-requisites
 
+1. Local development tools: git, terraform, a terminal.
+2. Understanding of OCI and its services, in particular the OCI Networking services.
```

Comment:
```suggestion
2. Understanding of Oracle Cloud Infrastructure (OCI) and its services, in particular the Virtual Cloud Networking (VCN) service.
```

_Do not commit_ this suggestion! Instead cut and paste it and commit it as yourself, otherwise the OCA Bot gets very confused.